### DISCUSSION TITLE
Capacitor build for Android and IOS

### DISCUSSION THREAD

**Discussion by leahfkd** [ionic-team/capacitor#5292] [Q&A]
Hello 
I am just getting started with capacitor, and I run some issues I couldn't understand why and or how to solve it. 
Maybe anyone here can help me.

These are my issues:

1. When I build IOS using `npx cap run ios` I am getting a build, but stylings are not reflected if I add it like ` <link href="static/css/app.ae0f4d7a0183da075d6813efb81018bd.css" rel="stylesheet">`, but when I tried testing it by adding `<style>body{background-color:red;}</style>` CSS in my index.html header, it works.

   _I also build this using xcode, stylings are reflected, and I didn't have the same issue. It works there._

2. **Cant build Android App**
I build it using `npx cap run android` and or android studio, but I keep getting the same problem `Task :app:mergeDebugAssets FAILED` ,  `Resource and asset merger: Duplicate resources`

I already tried a lot of suggestions I get from different communities with the same like problem, but couldn't fix it.

Hoping anyone from here can share their thoughts if they know what was the cause  or how to fix it. 

Thank you.

**Comment by ovictoraurelio** [comment]
Hello, I'm getting started with Capacitor to build a Android App and facing the same issue.

`Execution failed for task ':app:mergeDebugAssets'.`

One of multiple lines below:

`[public/assets/erudio-onboard-graph.a7061a0e.svg] /home/victor/projects/byndcloud/quoti-vue/android/app/src/main/assets/public/assets/erudio-onboard-graph.a7061a0e.svg.gz	[public/assets/erudio-onboard-graph.a7061a0e.svg] /home/victor/projects/byndcloud/quoti-vue/android/app/src/main/assets/public/assets/erudio-onboard-graph.a7061a0e.svg: Error: Duplicate resources`

**Comment by ovictoraurelio** [comment]
I have got a solution, to my problem.

I have figured out that my vuejs build with CompressionPlugin generates a file .html and .gz, a file .js and .gz... These files are the same but compressed or not compressed, disabling CompressionPlugin enables capacitor to build my APK.
